### PR TITLE
Suggest a method for the description and version

### PR DESCRIPTION
**Pull request by arielb2**
This patch suggests taking the information that is in "EGG-INFO/PKG-INFO"

### CODE REVIEW

**Comment by pypingou** [comment]
Thanks for the patch, but I'm curious, what is the added value?

**Comment by arielb2** [comment]
I think worth to mention it is that the information is not out of date placed in the setup.py

[setup.py](https://github.com/fedora-infra/pkgwat.cli/blob/develop/setup.py#L61)

**Comment by pypingou** [comment]
Hm indeed :(

For the version, there is a one-liner that could do it as well: https://github.com/fedora-infra/packagedb-cli/blob/master/pkgdb2client/__init__.py#L42

**Comment by arielb2** [comment]
you are right, I had thought to use only pkg_resources.get_distribution but only has three properties that are:
- version
- location
- name
and he was left outside the description.
But if you think it is better to use only version, I can change

**Comment by pypingou** [comment]
Looking at things, it seems the description is going to contain the entire readme right?

If that's so it likely does not bring much to someone who is skilled enough to have to ``pkgwat.cli.__description__`` within a python shell :)

File: pkgwat/cli/utils.py [hunk 1]
```diff
@@ -1,12 +0,0 @@
-import datetime
-import requests
-
-
-def _format_time(timestamp):
```
Comment (pypingou) [inline pkgwat/cli/utils.py]: was this intentional?

File: pkgwat/cli/utils.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import datetime
 import requests
 
-
```
Comment (pypingou) [inline pkgwat/cli/utils.py]: If we could keep the line in, pep8 would be happier :]

**Comment by arielb2** [comment]
done, thanks for your comments

**Comment by pypingou** [comment]
Looks good to me, thanks for the fix! :)

**Pull request merged**